crmd/notify.c

```diff
@@ -22,220 +22,16 @@
 #include <crm/pengine/rules.h>
 #include "notify.h"
 #include "crmd_messages.h"
+#include <crm/common/alerts_internal.h>
+#include <crm/common/iso8601_internal.h>
 
-static char *notify_script = NULL;
-static char *notify_target = NULL;
```

Comment:
notify_script and notify_target are only implemented for backward compatibility with an older CIB syntax that did not make it into a pacemaker release. So, we can keep notify_script and notify_target here in the crmd only. The library and attrd do not need to support them.